drivers/serial/uart_native_tty.c

```diff
@@ -143,6 +145,16 @@ static int native_tty_uart_poll_in(const struct device *dev, unsigned char *p_ch
 	return nsi_host_read(data->fd, p_char, 1) > 0 ? 0 : -1;
 }
 
+#ifdef CONFIG_UART_USE_RUNTIME_CONFIGURE
```

Comment:
This ifdef should not cover  native_tty_configure() as that one is always needed

Reply:
I see, I understand now.. btw stm32 uart driver is inconstent, by not using `configure` in its `init`…